parsers/IN_WE.py

```diff
@@ -29,65 +30,66 @@
 }
 
 KIND_MAPPING = {
-    "exchange": {"url": EXCHANGE_URL, "datetime_column": "lastUpdate"},
-    "consumption": {"url": CONSUMPTION_URL, "datetime_column": "current_datetime"},
+    "exchange": {
+        "url": EXCHANGE_URL,
+        "datetime_col": "lastUpdate",
+        "region_col": "Region_Name",
+    },
+    "consumption": {
+        "url": CONSUMPTION_URL,
+        "datetime_col": "current_datetime",
+        "region_col": "StateName",
+    },
 }
 
 
-def is_expected_downtime() -> bool:
-    current_day = datetime.now().weekday()
-    expected_outage_days = [5, 6, 0]  # Saturday, Sunday and Monday
-
-    if current_day in expected_outage_days:
-        return True
-
-    return False
-
-
 def get_date_range(dt: datetime):
+    """Returns 24 datetime objects for a given datetime's date, one for each hour."""
+    now_date_as_dt = datetime.combine(
+        datetime.now(ZONE_INFO).date(), datetime.min.time()
+    ).replace(tzinfo=ZONE_INFO)
```

Comment:
`datetime.now` shouldn't be used, but `dt`. Will fix this regression I've introduced.